infrastructure/docker/traffic_ops/Dockerfile

```diff
@@ -27,11 +28,16 @@
 #
 # docker run --name my-traffic-ops --hostname my-traffic-ops --net cdnet --publish 443:443 --env MYSQL_IP=my-traffic-ops-mysql --env MYSQL_PORT=3306 --env MYSQL_ROOT_PASS=secretrootpass --env MYSQL_TRAFFIC_OPS_PASS=supersecretpassword --env ADMIN_USER=superroot --env ADMIN_PASS=supersecreterpassward --env CERT_COUNTRY=US --env CERT_STATE=Colorado --env CERT_CITY=Denver --env CERT_COMPANY=NotComcast --env TRAFFIC_VAULT_PASS=marginallylesssecret --env DOMAIN=cdnet --detach traffic_ops:1.5.1
 
-FROM centos:7
+FROM centos/systemd
 MAINTAINER Dan Kirkwood
 
+RUN yum install -y https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-redhat96-9.6-3.noarch.rpm
+RUN yum install -y initscripts # needed for service command
+RUN yum install -y epel-release # needed for perl-Crypt-ScryptKDF
+RUN yum install -y perl-Crypt-ScryptKDF
 RUN yum install -y perl cpanminus perl-Test-CPAN-Meta
 RUN cpanm Carton
+RUN yum install -y perl-DBIx-Connector
 
 # Override RPM arg to use a different one using --build-arg RPM=...  Can be local file or http://...
 ARG RPM=http://traffic-control-cdn.net/downloads/1.7.0/RELEASE-1.7.0/traffic_ops-1.7.0-3908.5b77f60f.x86_64.rpm
```

Comment:
should probably change this to point to a non-existent local rpm file with a note to refer to the instructions on building it..   Having it point to an rpm that requires mysql may be confusing...

Reply:
Fixed.